xmake.lua

```diff
@@ -41,3 +41,4 @@ end)
 includes("deps")
 includes("UE4SS")
 includes("UVTD")
+includes("cxx_mods")
```

Comment:
Do we want to discuss where this should go, particularly if we are moving our install folders around?

Reply:
I don't believe we're moving any installation folders around, are we ? Am I missing something ?